src/test/java/core/basesyntax/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImplTest {
+    private ParserServiceImpl parser;
+
+    @BeforeEach
+    void setUp() {
+        parser = new ParserServiceImpl();
+    }
+
+    @Test
+    void parse_ok() {

```

Comment:
Please be attentive with method names it should match the convention:
`<methodUnderTest>_<state>_<expectedBehavior>`
Please check other similar cases.